internal/use_cases/create_pull_request.go

```diff
@@ -43,8 +48,50 @@ type CreatePullRequest struct {
 	BranchProvider          domain.BranchProvider
 }
 
+func (cpr *CreatePullRequest) normalizeTemplatePath(path string) (string, error) {
+	if path == "" {
+		return "", nil
+	}
+
+	if !filepath.IsAbs(path) {
+		repoRoot, err := cpr.Git.GetRepositoryRoot()
+		if err != nil {
+			return "", fmt.Errorf("failed to determine repository root: %w", err)
+		}
+		path = filepath.Join(repoRoot, path)
+	}
+
+	return path, nil
+}
+
+func (cpr *CreatePullRequest) validateTemplateFile() error {
```

Comment:
thought: maybe we can set this function like normalize: `func validateTemplateFile(template string)` and use it as a private function. With this we are not coupling the configuration to this function.